plugins/modules/ec2_asg_instance_refresh.py

```diff
@@ -0,0 +1,190 @@
+#!/usr/bin/python
+# Copyright: Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+DOCUMENTATION = '''
+---
+module: ec2_asg_instance_refresh
+version_added: 1.0.0
+short_description: Start or cancel an  ec2 Auto Scaling Group Instance Refresh in AWS
+description:
+  - Start or cancel an ec2 Auto Scaling Group Instance Refresh in AWS
+  - Can be used with ec2_asg_instance_refreshes_info to track the subsequent progress
+requirements: [ boto3 ]
+author: "Dan Khersonsky (@danquixote)"
+options:
+  state:
+    description:
+      - Desired state
+    type: str
+    required: true
+    choices: [ 'started', 'canceled' ]
+  name:
+    description:
+      - The name of the auto scaling group you are searching for.
+    type: str
+    required: true
+  strategy:
+    description:
+      - list of InstanceRefreshIds
```

Comment:
This doesn't look right

Reply:
fixed in f589805b81998e2538b418044aa09e69f8e50f50